rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -996,6 +996,54 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
```

Comment:
Why is this `protected` and not `public`? Where is this being called from?

Reply:
When creating the new method that would be called from updateCollections, it will become private, but I need it protected first to call it from the tests

Reply:
its being addressed in next pr